src/i18n/Streami18n.ts

```diff
@@ -179,6 +179,11 @@ export type TDateTimeParser = (input?: string | number | Date) => Dayjs.Dayjs |
 
 export type LanguageCallbackFn = (t: TFunction) => void;
 
+export type Translator = {
+  t: TFunction;
+  tDateTimeParser: TDateTimeParser;
+};
+
```

Comment:
could be refactored a bit since a similar type already exists `TranslationContextValue`

Reply:
Removing the duplicate. Thank you